src/pages/case-studies.js

```diff
@@ -12,11 +12,12 @@ import webCmsIcon from '../files/icons/web-cms-icon.svg';
 import workforceDevelopmentIcon from '../files/icons/workforce-dev-icon.svg';
 import SEO from '../components/seo';
 import { Link } from 'gatsby';
+import CaseStudyTeaser from '../components/case-study-teaser.js';
 
 const CaseStudyPage = () => {
   const data = useStaticQuery(graphql`
     {
-      allStrapiCaseStudy(sort: { fields: Sort_Order }) {
+      allStrapiCaseStudy(filter: {Featured: {eq: "True"}}) {
```

Comment:
@jaube05 this looks like it will cause the /case-studies page to only display featured case studies instead of all published case studies.